framework/libra-framework/sources/modified_source/transaction_fee.move

```diff
@@ -221,21 +221,26 @@ module diem_framework::transaction_fee {
     }
 
     /// root account can use system fees to pay multiple accounts, e.g. for Proof of Fee reward.
-    public fun vm_multi_pay_fee(vm: &signer, list: &vector<address>, amount: u64) acquires CollectedFeesPerBlock {
+    /// returns the actual amount that was transferred
+    public fun vm_multi_pay_fee(vm: &signer, list: &vector<address>, amount: u64): u64  acquires CollectedFeesPerBlock {
       system_addresses::assert_ol(vm);
+      let actual_transferred = 0;
```

Comment:
```suggestion
      let mut actual_transferred = 0;
      let mut i = 0;
```

Reply:
@hemulin  I don't think this will compile. must be Rust muscle memory ;)